kubectl-cloudflow/cmd/deploy.go

```diff
@@ -380,7 +407,12 @@ func createOrUpdateCloudflowApplication(
 func validateDeployCmdArgs(cmd *cobra.Command, args []string) error {
 
 	if len(args) < 1 || args[0] == "" {
-		return fmt.Errorf("please specify the full path to the Docker image containing the application. For example: 'docker-registry.mydomain.com/cloudflow/awesome-app:37-172e856'")
+		return fmt.Errorf("please specify the full path to the file containing the application CR")
+	}
+
+	crFile := args[0]
+	if !util.FileExists(crFile) {
+		return fmt.Errorf("Local file (%s) does not exist", crFile)
```

Comment:
O wait you do this here, minor: drop the "Local"

Reply:
done ..